contracts/Staking.sol

```diff
@@ -182,7 +166,7 @@ contract Staking is IERC20, IERC20Metadata {
 
     function updateRewardPool() public whenNotPaused {
         uint delta = block.timestamp - lastRewardTimestamp;
-        accamulatedRewardPerShare += totalAmount * delta * aprBasisPoints / 10000 / 31557600;
+        accumulatedRewardPerShare +=  delta * aprBasisPoints * calcDecimals / 10000 / 31557600;
```

Comment:
What is 31557600?

Reply:
Added constant representation of this value as secondsInYear